modules/qna/src/views/full/utils/qnaList.utils.ts

```diff
@@ -24,6 +24,7 @@ export interface Props {
   topicName: string
   contentLang: string
   defaultLanguage: string
+  getQnaCountByTopic: () => void
```

Comment:
Feels weird having a "getSomething" returning void. I'd expect that to return a list of topics with a number

Reply:
+1